pkgs/by-name/rl/rlama/package.nix

```diff
@@ -37,25 +38,29 @@ buildGoModule rec {
     makeWrapper
   ];
 
+  # Skip tests that require Ollama
+  doCheck = false;
```

Comment:
You are skipping ALL the tests here, can you please adjust the comment or make sure the tests are running?

Reply:
I will see what I can do but it requires ollama to pass all the tests.